addOns/webuipoc/src/main/pocs/reactWebUI/src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
Whats this file for?

Reply:
For writing unit tests, actually we've created this boilerplate using https://create-react-app.dev/ so everything comes packaged. 

We can/should write unit tests in this file in future PRs